### ISSUE TITLE
docs - null builder example is wrong

### ISSUE THREAD

**Issue by dpetzel** [mitchellh/packer#2770]
https://www.packer.io/docs/builders/null.html. Appears `ssh_host` should be `host`. Here what I get if I use the example as it exists in today.

```
null_builder output will be in this color.

2 error(s) occurred:

* Unknown configuration key: ssh_host
* host must be specified
```

changing the attribute to `host` in my template yielded success.

**Comment by caarlos0** [comment]
@dpetzel can you check #2794?

**Comment by markpeek** [comment]
Thank you for the issue. However, the current implementation uses ```ssh_host``` consistently via the helper/communicator interface. The switch from ```host``` to ```ssh_host``` occurred in d545431f9bc735f5200489917512da5230ec7418 with an error message fix in 750e09d51d0a3535e5d01c1926b28b1dc7a278b9.
